src/compiler/rust/canister/src/lib.rs

```diff
@@ -62,6 +62,12 @@ impl Storable for AzleStableBTreeMapValue {
     const BOUND: Bound = Bound::Unbounded;
 }
 
+type Hash = Option<Vec<u8>>;
+type Timestamp = u64;
+type BytesReceived = u64;
+
+const FILE_HASH_PATH: &str = ".config/azle/file_hashes.json";
```

Comment:
It seems like it is currently not supported to have Path::new() (which we would need to use join) with a const like this.
https://github.com/rust-lang/rust/pull/92930
https://stackoverflow.com/questions/76348617/why-is-it-not-possible-to-declare-a-static-or-constant-stdpathpath-object-in

Options:
1. Since we are currently only targeting canisters this file format should work for everything and we can just keep it like that for the time being.
2. We could just recreate this variable in each of the three places we use it.
3. We could do a hybrid where the parts of the path are strings and then at each use we make it into a path
4. We could make a function for it.
     5. I like that one, I think I'll try it

Reply:
Something like this perhaps?
```rust
pub fn get_file_hash_path() -> std::path::PathBuf {
    std::path::Path::new(".config")
        .join("azle")
        .join("file_hashes.json")
}
```